README.md

```diff
@@ -471,6 +492,30 @@ CMS are a practical way to setup a website with many features. CMS often come wi
 
 See https://staticsitegenerators.net and https://www.staticgen.com
 
+
+## Statistics
+*Analytics software.*
+
+* [AWStats](http://www.awstats.org/) - Generates web, streaming, ftp or mail server statistics graphically.
+* [GoAccess](http://goaccess.io/) - Real-time web log analyzer and interactive viewer that runs in a terminal.
+* [Open Web Analytics](http://www.openwebanalytics.com/) - Add web analytics to websites using JS, PHP or REST APIs.
+* [Piwik](http://piwik.org/) - Web analytics application.
+* [Webalizer](http://www.webalizer.org/) - Fast web server log file analysis.
+
+
+## Ticketing systems
```

Comment:
Can you move this to the existing Bug and Task Trackers subsection under Software Development?